omnibus/resources/agent/msi/assets/sitedlg.wxi

```diff
@@ -0,0 +1,32 @@
+<Include>
```

Comment:
That is probably possible, but it kind of goes against omnibus: when building a project, the only resources folder omnibus sees (ie. the value of `project.resources_path`) is `resources/<project_name>`.
That means we would have to define the file somewhere and add a manual step before / during the omnibus build (before the packaging step) to copy this file in the right place for the build.

Reply:
That wouldn't work because the packaging is done in the omnibus staging directory, and omnibus only copies the contents of `resources/<project>/<packager>/assets` + the source, bundle and localization files in there (so `..\CommonDialogs` wouldn't exist).